Before-Using-Agoric.md

```diff
@@ -0,0 +1,79 @@
+
+# Before Using Agoric Software
+
+Before working with the Agoric CLI, Zoe, and other Agoric tools and
+software, you need to install the following. 
+
+<table border="1">
+  <tbody>
+  <th><b>Step</b></th>
+  <th><b>Action</b></th>
+  <th><b>Explanation</b></th>
+  <tr>
+    <td><center>1</center></td>
+    <td><a href="https://nodejs.org/">Install Node.js</a>, version 10.16.0 or higher</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td><center>2</center></td>
+    <td><a href="https://classic.yarnpkg.com/en/docs/install#mac-stable">Install Yarn 1</a></td>
+    <td>Yarn is a package manager for your code and lets developers
+  share code with others. Code is shared via a <i>package</i> (also
+  called a <i>module</i>) that contains all shared code and a
+  <code>package.json</code> file describing the package. The link takes you to 
+  the Yarn install page, where you first select what operating system you want to
+  install on. Your selection changes the page's content to give install instructions for that 
+  OS and links to the needed downloads.</td>
+  </tr>
+  <tr>
+    <td><center>3</center></td>
+    <td>Open a shell. The rest of this table's Actions take place in
+  this shell.</td>
+    <td>A terminal on Macs; see 
+      <b>Finder&gt;Applications&gt;Utilities&gt;terminal</b></td>
+  </tr>
+  <tr>
+    <td><center>4</center></td>
+    <td>Run <code>git clone
+      https://github.com/Agoric/agoric-sdk</code></td> 
+    <td>Get the latest Agoric SDK from the Agoric GitHub
+  repository. It goes into the (created, if necessary)
+      <code>agoric-sdk</code> sub-directory of your home directory.</td>
+  </tr>
+  <tr>
+    <td><center>5</center></td>
+    <td>Run <code>cd ~/agoric-sdk</code></td>
+    <td>Change to the <code>agoric-sdk</code> subdirectory in your home
+  directory.</td>
+  </tr>
+  <tr>
+    <td><center>6</center></td>
+    <td>Run <code>yarn install</code></td>
+    <td>Install NPM dependencies.</td>
+  </tr>
+  <tr>
+    <td><center>7</center></td>
+    <td>Run <code>yarn build</td>
+    <td>Build sources that need compiling. <b>Note:</b>
+  <code>build</code> is not a standard <code>yarn</code> parameter,
+  but one installed with the Agoric SDK.</td>
+  </tr>
+  <tr>
+    <td><center>8</center></td>
+    <td>Install the Agoric CLI by running: <code>yarn link-cli 
+      &lt;<i>agoric-cli's install directory</i>&gt;</td>
+    <td>Select a directory to install the Agoric CLI in. For example,
+      <code>yarn link-cli  
```

Comment:
Do we expect this step and the next step to change?

Reply:
@tyg The parameter is not `agoric-cli's install directory`, it is `filename of new cli script to create`.  I unfortunately do not have a really good way of explaining this.

Remind me to rename (or at least create an alias of) `link-cli` to `create-sdk-cli` and update these docs at the same time.

Reply:
Issue https://github.com/Agoric/agoric-sdk/issues/931 created on agoric-sdk.